packages/cmus/build.sh

```diff
@@ -3,14 +3,23 @@ TERMUX_PKG_DESCRIPTION="Small, fast and powerful console music player"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="2.12.0"
-TERMUX_PKG_REVISION="1"
+TERMUX_PKG_REVISION="2"
 TERMUX_PKG_DEPENDS="ffmpeg, libandroid-support, libflac, libiconv, libmad, libmodplug, libvorbis, libwavpack, ncurses, opusfile, pulseaudio"
 TERMUX_PKG_SRCURL=https://github.com/cmus/cmus/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=44b96cd5f84b0d84c33097c48454232d5e6a19cd33b9b6503ba9c13b6686bfc7
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true
 
 termux_step_pre_configure() {
+	# cherry-pick patches
+	local sha commits=(
+		56f21a455bc611d879ad5384ca26c154bd59a34d # ensure the aaudio buffer is at least 80ms (cmus/cmus#1386)
```

Comment:
This is 40 characters long, so it's obviously not the full SHA256 hash.
If you're just using the short-hash for the commit the first 7 chars should be sufficient.
Either use the full commit hash or use the proper short-hash.

Please also link to the PR with a full link.
```suggestion
		56f21a455bc611d879ad5384ca26c154bd59a34d # ensure the aaudio buffer is at least 80ms (https://github.com/cmus/cmus/pull/1386)
```

I also can't help but notice that this is your own upstream PR, from about an hour ago.
I'd like to hear back from upstream about their opinion on this before making any decision regarding this PR.

Reply:
>  Either use the full commit hash or use the proper short-hash.

Ah, will take a look. I copied that from the GitHub website.

> I also can't help but notice that this is your own upstream PR, from about an hour ago.

It's merged.